packages/api/src/db/user.ts

```diff
@@ -117,8 +117,8 @@ export interface DeleteUserSubscriptionArgs {
 }
 
 export interface PaymentProviderCustomer {
-  readonly id: string
-  readonly createdAt: Date
```

Comment:
Is `createdAt` not relevant anymore ?

Reply:
Yeah i added it in the past but realized now that it was premature optimization and we don't need it.